modules/blazium_sdk/lobby/lobby_client.cpp

```diff
@@ -91,12 +103,13 @@ String LobbyClient::_increment_counter() {
 	return String::num(_counter++);
 }
 
-Ref<LobbyClient::CreateLobbyResponse> LobbyClient::create_lobby(int max_players, const String &password) {
+Ref<LobbyClient::CreateLobbyResponse> LobbyClient::create_lobby(String lobby_name, int max_players, const String &password) {
```

Comment:
Ref<LobbyClient::CreateLobbyResponse> LobbyClient::create_lobby(const String &p_lobby_name, int p_max_players, const String &p_password) {

It's better to follow the godot c++ style.

`p_` prefix before function's parameter.

`const String &p_string`
`const Vector2 &p_vector`